pulsar-io/hdfs2/pom.xml

```diff
@@ -63,6 +63,23 @@
         <groupId>org.apache.nifi</groupId>
         <artifactId>nifi-nar-maven-plugin</artifactId>
       </plugin>
+      <plugin>
+        <groupId>com.github.spotbugs</groupId>
+        <artifactId>spotbugs-maven-plugin</artifactId>
+        <version>${spotbugs-maven-plugin.version}</version>
+        <configuration>
+            <excludeFilterFile>${basedir}/src/main/resources/findbugsExclude.xml</excludeFilterFile>
```

Comment:
It seems we don't need to set `excludeFilterFile` since it's empty.

Reply:
I did this because I referenced other related pr, such as #8842 and #8903.  
Do I need to delete this file?

Reply:
@Jason918 Looking forward to your confirmation and reply.